### PR TITLE
Fixes error in designer #13032

### PR DESCRIPTION
**Pull request by Achilles-96**
Signed-off-by: Raghuram Vadapalli <raghuram.vadapalli@research.iiit.ac.in>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

File: js/pmd/move.js [hunk 1]
```diff
@@ -1298,9 +1305,18 @@ function Relation_lines_invert()
     Re_load();
 }
 
+function Get_html_string(str)
+{
+    var parser = new DOMParser;
+    var dom = parser.parseFromString('<!doctype html><body>' + str,'text/html');
```
Comment (nijel) [inline js/pmd/move.js]: This seems to open door to XSS.

**Review by nijel — CHANGES_REQUESTED** [review]
I don't think this is good approach. We most likely need to encode the values rather than doing some transforms on it later.

**Comment by nijel** [comment]
Sorry, but I still think this is wrong approach:

- trying to treat arbitrary value (database or table name) as identifier will never work reliably
- using something like md5 or similar hash is probably better approach

**Comment by Achilles-96** [comment]
I am sorry I forgot about this. I will try to send a patch where tables will have their md5 sums as their ids. I already used that approach in my other work - #13342 . I will do this as soon as I am free if no one takes this up.

**Pull request closed without merging**